### PR TITLE
Close the report file descriptor before creating zip file.

### PR DESCRIPTION
**Pull request by douglashall**
This should fix a problem we were seeing where the final report file had records truncated from the end.

### CODE REVIEW

**Comment by douglashall** [comment]
@georgebabey @brittneyexline Reminder to review this. It would be great if I could merge this before the 11am run of the Jenkins job.

File: enterprise_reporting/reporter.py [hunk 1]
```diff
@@ -110,21 +115,6 @@ def send_enterprise_report(self):
...
-        """
-        data_report_file_name = self.REPORT_FILE_NAME_FORMAT.format(
-            path=self.FILE_WRITE_DIRECTORY,
-            enterprise_id=self.reporting_config['enterprise_customer']['uuid'],
-            date=datetime.datetime.now().strftime("%Y-%m-%d"),
-            extension='csv',
-        )
-        data_report_file = open(data_report_file_name, 'w')  # pylint: disable=open-builtin
-        data_report_file_writer = csv.writer(data_report_file)
-        data_report_file_writer.writerow(self.VERTICA_QUERY_FIELDS)
```
Comment (brittneyexline) [inline enterprise_reporting/reporter.py:125]: we still need to write the first row with the field names above

**Review by georgebabey — APPROVED** [review]
Looks good :shipit:

File: enterprise_reporting/reporter.py (hunk 1 above)
Comment (douglashall) [reply enterprise_reporting/reporter.py:125]: Good catch. I lost that row in the copy/paste.

Comment (douglashall) [reply enterprise_reporting/reporter.py:125]: Should be all set now.

**Pull request merged**